httpclient/httpclient.go

```diff
@@ -471,16 +458,12 @@ func (c *Client) retryRequest(ctx context.Context, name string, r Request, newRe
 
 		res, err := c.httpClient.Do(req)
 		if err != nil {
-			if errors.Is(err, context.DeadlineExceeded) {
-				requestClose = true
-			}
 			// url errors repeat the method and url which clutters metrics and logging
 			e := &url.Error{}
 			if errors.As(err, &e) {
 				err = e.Err
 			}
-			return fmt.Errorf("call: %s %s failed with: %w after %d attempt(s)",
-				req.Method, r.route, err, attemptCounter)
+			return fmt.Errorf("httpclient do: %w", err)
```

Comment:
This is likely to break a bunch of tests in downstream projects, just so you're aware.